HLTrigger/special/plugins/HLTMuonPointingFilter.cc

```diff
@@ -104,8 +116,17 @@ bool HLTMuonPointingFilter::filter(edm::Event& event, const edm::EventSetup& eve
     if ( tsosAtCyl.isValid() ) {
       LogDebug("HLTMuonPointing") << " extrap TSOS " << tsosAtCyl;
```

Comment:
I think it would make more sense to move all `LogDebug`s here, rather than inside the `if`s